nix/docker.nix

```diff
@@ -68,14 +62,16 @@ in
     fromImage = baseImage;
     contents = [
       cardano-node
-      startScript
     ];
     created = "now";   # Set creation date to build time. Breaks reproducibility
     extraCommands = ''
       mkdir -p data config
     '';
     config = {
-      EntryPoint = [ "start-cardano-node" ];
+      EntryPoint = [ "${cardano-node}/bin/cardano-node" "--config"
+      "/config/config.yaml" "--database-path" "/data.db" "--genesis-file"
+      "/config/genesis.json" "--host-addr" "127001" "--port" "3001"
```

Comment:
```suggestion
      "/config/genesis.json" "--host-addr" "127.0.0.1" "--port" "3001"
```

Reply:
I clearly need to pay more scrutising to your pastes :rofl: 

Resolved in commit [51b43a7](https://github.com/input-output-hk/cardano-node/pull/525/commits/51b43a73362eec741d4aa25f08ca5fade9b92e42)